pandas/tests/test_generic.py

```diff
@@ -881,6 +881,13 @@ def testit(index, check_index_type=True, check_categorical=True):
         self.assertIsInstance(result, DataArray)
         assert_series_equal(result.to_series(), s)
 
+    def test_empty_method(self):
+        s_empty = pd.Series()
+        tm.assert_equal(s_empty.empty, True)
+
+        for full_series in [pd.Series([1]), pd.Series(index=[1])]:
```

Comment:
look in tests/frame/test_misc_api.py (and series). prob some panel tests already.

you can add cases to there if needed